conf/base.config

```diff
@@ -57,7 +57,7 @@ process {
         cache = false
     }
     withName:TDF2MZML {
-        cpus   = { check_max( 10    * task.attempt, 'cpus'    ) }
+        cpus   = { check_max( 1                   , 'cpus'    ) }
```

Comment:
I think the `* task.attempt` must stay
(see sarek https://github.com/nf-core/sarek/blob/f034b737630972e90aeae851e236f9d4292b9a4f/conf/base.config#L12C1-L12C1)